services/export/tests/unit/shared/helpers/ready.test.js

```diff
@@ -0,0 +1,316 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2024 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const { resolve } = require('path');
+
+const mockedRequestFilePath = resolve(`${__dirname}/../../../../shared/helpers/request`);
+const mockedHelpersPath = resolve(`${__dirname}/../../../../shared/helpers`);
+
+describe('Test getIndexStatus method', () => {
+	it('should return the index status', async () => {
+		const mockIndexStatus = {
+			data: {
+				genesisHeight: 23390992,
+				lastBlockHeight: 23827824,
+				lastIndexedBlockHeight: 23395523,
+				chainLength: 436833,
+				numBlocksIndexed: 4532,
+				percentageIndexed: 1.03,
+				isIndexingInProgress: true,
+			},
+			meta: {
+				lastUpdate: 1706251273,
+			},
+		};
+
+		jest.mock(mockedRequestFilePath, () => {
+			const actual = jest.requireActual(mockedRequestFilePath);
+			return {
+				...actual,
+				requestIndexer() {
+					return mockIndexStatus;
+				},
+			};
+		});
+
+		const { getIndexStatus } = require('../../../../shared/helpers/ready');
+		const response = await getIndexStatus();
+
+		expect(response).toEqual(mockIndexStatus);
+	});
+});
+
+describe('Test checkIfIndexReadyForInterval method', () => {
+	describe('when indexing is at 100 percent', () => {
+		beforeEach(() => {
+			jest.resetModules();
```

Comment:
Not important but instead of resetting Jest modules in every tests, we can also move it outside the describe
```
beforeEach(() => jest.resetModules());
```